draft-ietf-tls-tls13.md

```diff
@@ -2864,15 +2841,22 @@ The idea is just to minimize exposure.]]
 server_key
 : The long-term DH key that is being established for this configuration.
 
-early_data_allowed
-: Whether the client may send data in its first flight (see {{early-data-indication}}).
+early_data_type
+: The type of 0-RTT handshake that this configuration is to be used
+for (see {{early-data-indication}}). If "client_authentication"
+or "client_authentication_and_data", then the client should select
+the certificate for future handshakes based on the CertificateRequest
+parameters supplied in this handshake.
```

Comment:
I think that this needs a lot more detail.  If the server requests client authentication in the first connection and the client does not provide a certificate, is the client permitted to connect without authentication in future?  Are they expected to copy their choice, or can they make the new connection with authentication?  What happens if the server does not request a certificate in the first connection but provides a configuration that permits client authentication?